test/postgres.test.coffee

```diff
@@ -79,6 +89,11 @@ test['Postgres flavour'] =
       toString: ->
         assert.same @del.toString(), 'DELETE FROM table WHERE (field = 1) RETURNING field'
 
+    '>> from(table).where(field = 1).with(alias, table)':
+      beforeEach: -> @del.from('table').where('field = 1').with('alias', squel.select().from('table'))
+      toString: ->
```

Comment:
Please add a `toParam()` test too, and the `with` query should also use params, e.g. `select().from('table').where('a = ?', 5)`